pkg/api/admin/openshiftversion.go

```diff
@@ -0,0 +1,17 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// OpenShiftVersionList represents a list of OpenShift clusters.
```

Comment:
nit: this isn't returning a 'cluster' but the versions of openshift that are available, can we reword this a bit?